microsoft-edge/webview2/webview2-api-reference.md

```diff
@@ -29,6 +29,9 @@ WebView2 is available for the following frameworks (or platforms) and programmin
 
 * [Win32/C++](/microsoft-edge/webview2/reference/win32/index)
 
+Additionally, WebView2 exposes a set of JavaScript APIs to the HTML documents running within WebView2, mostly to facilitate communication with the host application:
```

Comment:
@peiche-jessica - 
1. See the new tab-set format design at live site:  https://learn.microsoft.com/microsoft-edge/webview2/webview2-api-reference
2. Update your local "main" branch from server upstream main.
3. Merge main into this PR's branch.
4. Push commit.  
5. Add a 4th, rightmost tab if that layout makes sense to do.  So, add a 4th tab-heading like `##### [JavaScript](#tab/js)` just above the `---` line.  Keep blank lines above and below the tab-set terminator `---` line.

Reply:
I'd lean towards keeping "JavaScript APIs" separate from the tab set as they are meant to work together with the native WebView2 APIs (in the tab set).

Reply:
The rendered page with option 2

![image](https://user-images.githubusercontent.com/55165503/217645919-5be5936d-d3a1-4537-8998-611d793ebd79.png)